bp_common/test/Makefile.tests

```diff
@@ -45,6 +46,9 @@ spec_build:
 riscvdv_build:
 	$(MAKE) -C $(riscvdv_dir)
 
+linux_build: $(fw_payload)
```

Comment:
Need to add to a suite folder or the build system won't be able to find it. Just linux/linux.riscv I guess?

Reply:
Which part of the build system?

Reply:
make sim.v

Reply:
Oh I see. We are categorizing the tests. Do I need to just need to change th cp destination or should I also specify somewhere that which suite linux belongs to?

Reply:
You can just change the cp destination.  If you add to bp_common/test/Makefile.frag, it can autorebuild, but I think it's probably better to require explicit builds of linux since it's so intensive